File: Hoops.Core/Entities/SponsorProfile.cs
using System;
using System.Collections.Generic;

namespace Hoops.Core.Entities
{
    public partial class SponsorProfile
    {
        public int CompanyId { get; set; }
        public int SponsorProfileId { get; set; }
        public int? HouseId { get; set; }
        public string ContactName { get; set; }
        public string SpoName { get; set; }
        public string Email { get; set; }
        public string Url { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string Zip { get; set; }
        public string Phone { get; set; }
        public string TypeOfBuss { get; set; }
        public bool? ShowAd { get; set; }
        public DateTime? AdExpiration { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string CreatedUser { get; set; }
    }
}

File: hoops.ui/src/domain/sponsor-profile.ts
export interface SponsorProfile {
  sponsorProfileId: number;
  companyId: number;
  createdDate?: Date | string | null;
  createdUser?: number | null;
  modifiedDate?: Date | string | null;
  modifiedUser?: number | null;
  houseId: number | null;
  spoName: string;
  contactName: string;
  email: string;
  phone: string;
  url: string;
  address: string;
  city: string;
  state: string;
  zip: string;
  typeOfBuss: string;
  showAd: boolean;
  adExpiration: Date | string | null;
}

export interface SponsorSeason {
  sponsorId: number;
  sponsorProfileId: number;
  seasonId: number | null;
  seasonDescription: string;
  shirtName: string;
  shirtSize: string;
  spoAmount: number | null;
  feeId: number | null;
  mailCheck: boolean | null;
  adExpiration: Date | string | null;
}

export interface SponsorPayment {
  paymentId: number;
  sponsorProfileId: number;
  createdDate?: Date | string | null;
  createdUser?: number | null;
  modifiedDate?: Date | string | null;
  modifiedUser?: number | null;
  amount: number;
  paymentType: string;
  transactionDate: Date | string | null;
  transactionNumber: string;
  memo: string;
}

export interface SponsorFee {
  sponsorFeeId: number;
  feeName: string;
  amount: number;
}

export interface SponsorListItem {
  id: number;
  sponsorProfileId: number;
  spoName: string;
  contactName: string;
  email: string;
  phone: string;
  lastSeasonId: number | null;
  lastSeasonDescription: string | null;
}
